Data/Base/Shaders/Materials/SkeletonVisualization.ezShader

```diff
@@ -95,41 +95,40 @@ void main( triangle GS_IN input[3], inout TriangleStream<GS_OUT> outputStream )
 Texture2D BaseTexture;
 SamplerState BaseTexture_AutoSampler;
 
-float edgeFactor(PS_IN Input)
+float edgeFactor()
```

Comment:
was this shader previously just broken ?

Reply:
Not all permutations, the permutations in which the functions were not called were working. It could be that the other permutations are not used in practice but when compiling all via the shadercompiler tool it would fail. I don't know under which circumstances which permutations are used so I decided to fix the permutations instead of reducing the permutation space.